pkg/server/rds/rds.go

```diff
@@ -31,7 +31,7 @@ func (p *RDSPlugin) DatabasePlugin() plugin.DatabasePlugin {
 }
 
 func (p *RDSPlugin) Dsn(userName string, password string, db *atlas.Database, s *atlas.DatabaseServer) string {
-	return ""
+	return p.DatabasePlugin().Dsn(userName, password, nil, s)
```

Comment:
are we considering dbHost while creating DSN and DSNSecret to be used by database resource ?

Reply:
dsn for database resources also will be created by database server resource we should not allow to use user dsn in database resource.

dbHost will be used to create external service. DSN and DSNSecret will use the service name not the dbHost directly.